### PR TITLE
refactor: number formatting and tooltip placement

### LINKED ISSUE #757: Improvement: format block heights
Currently the blockheight is only formatted in the heading part of the page that also shows the market cap. We should format the blockheight in other places too, e.g. here https://explorer.ark.io/block/8313be6f8cd555f2ef4580521ca6311392bbccf3bf39c179a4151913ecc2d7ed , to make it easier to read.

### PR DESCRIPTION
**Pull request by dated**
## Summary

Closes #757.

## Checklist

- [ ] Documentation _(if necessary)_
- [x] Tests _(if necessary)_
- [x] Ready to be merged

### CODE REVIEW

**Comment by ghost** [comment]
**Thanks for submitting this pull request!** A maintainer will review this in the next few days and explicitly select labels so you know what's going on.

_If no reviewer appears after a week, a reminder will be sent out._

**Review by ItsANameToo — APPROVED** [review]

**Comment by ghost** [comment]
A contributor has approved this PR. A maintainer will merge this PR shortly. If it shouldn't be merged yet, please leave a comment saying so and we'll wait.

_Thank you for your contribution!_

**Comment by ghost** [comment]
Your pull request has been merged and marked as tier 4. _It will earn you $20 USD._

File: resources/views/livewire/price-stats.blade.php [hunk 1]
```diff
@@ -49,8 +48,8 @@ class="ml-6"
             x-init="init"
             @toggle-dark-mode.window="toggleDarkMode"
         >
-            <div class="block" wire:ignore style="width: 120px; height: 40px;">
```
Comment (ItsANameToo) [inline resources/views/livewire/price-stats.blade.php:52]: @alfonsobries I removed these instances as they didn't seem to do anything, did you have any reason to add them on both the divs?

File: resources/views/livewire/price-stats.blade.php [hunk 2]
```diff
@@ -49,8 +48,8 @@ class="ml-6"
...
-                <canvas x-ref="chart" class="w-full h-full" ></canvas>
+            <div class="block" wire:ignore>
```
Comment (leMaur) [inline resources/views/livewire/price-stats.blade.php:51]: a `div` is a block by default

Comment (ItsANameToo) [reply resources/views/livewire/price-stats.blade.php:51]: That's very true

**Pull request merged**